File: src/main/java/no/nav/veilarboppfolging/repository/entity/VeilederTilordningHistorikkEntity.java
package no.nav.veilarboppfolging.repository.entity;

import lombok.Builder;
import lombok.Value;

import java.time.ZonedDateTime;

@Value
@Builder
public class VeilederTilordningHistorikkEntity {
    String veileder;
    ZonedDateTime sistTilordnet;
    String tilordnetAvVeileder;
}

File: src/main/kotlin/no/nav/veilarboppfolging/repository/entity/VeilederTilordningHistorikkEntity.kt
package no.nav.veilarboppfolging.repository.entity

import java.time.ZonedDateTime


data class VeilederTilordningHistorikkEntity(
    val veileder: String,
    val sistTilordnet: ZonedDateTime,
    val tilordnetAvVeileder: String?,
)
